Resources/Prototypes/_Goobstation/Changeling/Entities/Clothing/OuterClothing/hardsuits.yml

```
@@ -0,0 +1,34 @@
+    modifiers:
+      coefficients:
+        Blunt: 0.95
+        Slash: 0.95
+        Piercing: 1
+        Heat: 0.5
+  - type: ClothingSpeedModifier
+    walkModifier: 0.9
+    sprintModifier: 0.9
+  - type: HeldSpeedModifier
+  - type: Unremoveable
+  - type: ModifyDelayedKnockdown # Goobstation
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)